### DISCUSSION TITLE
问题反馈

### DISCUSSION THREAD

**Discussion by xiaobao-810216** [ZuodaoTech/everyone-can-use-english#988] [General]
你好，本地转写的时候能不能直接支持字幕导入啊， 这样就很方便了，现在本地转写，Whipser太慢了，很容易出现bug，就转写失败了，谢谢。

**Comment by an-lee** [comment]
不是已经支持了吗？

![image](https://github.com/user-attachments/assets/e747642e-1312-42f3-9187-a02d1ae1aeb6)
